runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java

```diff
@@ -276,4 +276,15 @@ default int infoBoxSize()
 	{
 		return 35;
 	}
+
+	@ConfigItem(
+		keyName = "tooltipPosition",
+		name = "Tooltip Position",
+		description = "Configures whether to show the tooltip above or under the cursor.",
+		position = 35
```

Comment:
The config items should be sorted by position

Reply:
Sorted